src/App.tsx

```diff
@@ -1,173 +1,18 @@
-import { Loader } from './components/Loader';
+import { Outlet } from 'react-router-dom';
 
 import './App.scss';
+import { Navbar } from './components/Navbar/Navbar';
 
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
+export const App = () => {
+  return (
+    <div data-cy="app">
+      <Navbar />
```

Comment:
Suggestion: You can create layout component with Navbar as static part and <Outlet > as dynamic.